src/XliffTasks/Model/ResxDocument.cs

```diff
@@ -52,5 +54,23 @@ protected override IEnumerable<TranslatableNode> GetTranslatableNodes()
                     element: valueElement);
             }
         }
+
+        public override void RewriteRelativePathsToAbsolute(string sourceFullPath)
+        {
+            foreach (var node in Document.Descendants("data"))
+            {
+                if (node.Attribute("type")?.Value == "System.Resources.ResXFileRef, System.Windows.Forms")
+                {
+                    var valueNodeOfFileRef = node.Element("value");
+                    var splittedRelativePathAndSerializedType = valueNodeOfFileRef.Value.Split(';');
```

Comment:
splitted is not a word. `splitRelativePathAndSerializedType`.

Reply:
fixed